samples/hello_world/prj.conf

```diff
@@ -1 +1,4 @@
 # nothing here
+CONFIG_LOG=y
```

Comment:
You're already on it I'm sure, but please create separate samples and / or tests.